src/server.js

```diff
@@ -80,6 +80,16 @@ function allRequests(req, res, next) {
 	return next();
 }
 
+// Information to populate the About section in UI, either from npm or from git
+try {
+	var gitCommit = require("child_process")
+		.execSync("git rev-parse --short HEAD") // Returns hash of current commit
+		.toString()
+		.trim();
+} catch (e) {
+	// Not a git repository or git is not installed: treat it as npm release
```

Comment:
> If git is not installed, then we will falsely report that this is an npm release.

Yeah, let's just ignore that. If someone has somehow got the git repository without having git installed, then we don't really care, as it's such a small thing anyway.

I think this solution is fine for what you are trying to do. The failure cases are so rare, and it's such a small thing (it'll just show it as a release rather than git) that it really doesn't matter.